src/fermilib/utils/_plane_wave_hamiltonian_test.py

```diff
@@ -27,11 +27,29 @@
     plane_wave_u_operator,
     dual_basis_u_operator,
     jordan_wigner_dual_basis_hamiltonian,
+    wigner_seitz_length_scale,
 )
 
 
 class PlaneWaveHamiltonianTest(unittest.TestCase):
```

Comment:
We have finished this discussion already before release and concluded that pytest would be nicer, therefore pytest is being used since the first release on Travis. 

 One argument was the smaller boilerplate required, a lot more functionality of sharing fixtures, ...